tests/beaker/lib/lvm_helper.rb

```diff
@@ -33,6 +33,23 @@ def verify_if_created?(agent, resource_type, resource_name, vg=nil, properties=n
           assert_match(/#{properties}/, result.stdout, 'Unexpected error was detected')
         end
       end
+    when 'aix_physical_volume'
+      on(agent, "lspv #{resource_name}") do |result|
+        assert_match(/Physical volume #{resource_name} is not assigned to/, result.stdout, 'Unexpected error was detected')
+      end
+    when 'aix_volume_group'
+      on(agent, "lsvg") do |result|
+        #assert_match(/#{resource_name}\s+^a-z0-9$\s+#{vg}/, result.stdout, 'Unexpected error was detected')
+        assert_match(/#{resource_name}/, result.stdout, 'Unexpected error was detected')
+      end
+    when 'aix_logical_volume'
+      fail_test "Error: missing volume group that the logical volume is associated with" unless vg
```

Comment:
The method throws exceptions everywhere else except here. I think you should change this to an exception so it can be caught by the caller.